pixi.toml

```diff
@@ -18,3 +45,44 @@ rasterio = ">=1.4.3,<2"
 shapely = ">=2.0.6,<3"
 matplotlib = ">=3.9.3,<4"
 tqdm = ">=4.67.1,<5"
+
+[feature.dev.tasks]
+docs = "jb build doc"
+configure = { cmd = [ "cmake",
+    "-G", "Ninja", 
+    "-S", ".",
+    "-B", ".build",
+    "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+    "-DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX",
+    "-DWITH_TESTS=ON",
+    "-DCMAKE_VERBOSE_MAKEFILE=OFF",
+    "-DCMAKE_LIBRARY_PATH:FILEPATH=$CONDA_PREFIX/Library/lib",
+    "-DCMAKE_INCLUDE_PATH:FILEPATH=$CONDA_PREFIX/Library/include",
+    "-DOPENSSL_ROOT_DIR=$CONDA_PREFIX/Library",
+    "-DPython3_ROOT_DIR:FILEPATH=$CONDA_PREFIX",
+    "-DPython3_FIND_STRATEGY=LOCATION",
+    "-DBUILD_PLUGIN_CPD=OFF",
```

Comment:
This will take just a little more work. First attempt with enabling it (adding `cpd` and `fgt` as dependencies) caused some strange errors that will likely require a little upstream work. The `cpd-config.cmake` file is being configured such that `WITH_FGT` contains the LIBRARY_PREFIX path and not just ON/OFF. Maybe that's legit, but there is a mixing of forward and backward slashes in that path that are tripping up CMake.

Reply:
oh, right, I forgot about that. No need to hold things up for it.